control-plane/grpc/proto/v1/volume/volume.proto

```diff
@@ -150,6 +150,8 @@ message GetVolumesRequest {
   }
   // pagination to allow for multiple requests to get all volumes
   common.Pagination pagination = 2;
+  // ignore 404 not found errors
```

Comment:
Question: Why is this needed?

Reply:
While debugging a cluster I noticed we end up with a ton of error messages which are actually not an error because on initial creation we do Get Volume xyz to find out if it exists or not, and first time it won't exist.
This prevents the error..
Usually we wouldn't need this flag but it seems we use the same grpc call to get 1 or many, so that's why it's needed as we cannot differentiate..